### DISCUSSION TITLE
image-optimizer.ts  swallows error

### DISCUSSION THREAD

**Discussion by szogun1987** [vercel/next.js#43476] [Help]
### What version of Next.js are you using?

10.2.2

### What version of Node.js are you using?

14.16 and 12.18

### What browser are you using?

Brave and Chrome

### What operating system are you using?

Windows & alpine

### How are you deploying your application?

Dokcer

### Describe the Bug

For some reason, my dockerized NextJS application crashes if I use the Image component. I suppose that it is related to writing images to the cache, but there are no logs that confirm/deny it.
I believe that it is related to catch at line number 340 in the image-optimizer.ts file
Could you add some console.error or console.warn there?

### Expected Behavior

I would like to see anything helpful in logs.

### To Reproduce

I build my app on my windows machine first (build requires access to database) and then I use following dockerfile to build an image.
```
FROM node:12.18.3-alpine

ENV PORT 3000

#RUN mkdir -p /usr/src/app/.next

WORKDIR /app

# Install app dependencies
COPY package*.json /app/
RUN npm install

RUN addgroup -g 1001 -S nodejs
RUN adduser -S nextjs -u 1001

# Bundle app source
COPY ./ /app/

RUN chown nextjs:nodejs /app/.next

ENV NODE_ENV production
ENV NEXT_TELEMETRY_DISABLED 1

USER nextjs

EXPOSE 3000

CMD [ "npx", "next", "start"]
```

**Comment by jankaifer** [comment]
This is probably issue with your docker setup so I'll move this to discussions. If you think that there is a next.js issue then please create a minimal reproduction with https://github.com/vercel/next.js/tree/canary/examples/reproduction-template and include your docker file there.
